python/tests/test_account.py

```diff
@@ -1970,6 +1970,42 @@ def test_set_get_group_image(self, acfactory, data, lp):
         assert msg_back.chat == chat
         assert chat.get_profile_image() is None
 
+    def test_fetch_deleted_msg(self, acfactory, lp):
```

Comment:
Add some `"""doc comment"""` here telling that it is a regression test for messages with `\Deleted` flag downloaded multiple times, otherwise we will forget why there is a test for such a thing at all, especially if it is moved to another file and `git blame` no longer works.